### PR TITLE
Add Docker install instructions

### PR DESCRIPTION
**Pull request by ellwoodb**
Hey @chrivers, me again. Just got to writing some instructions for the Docker installation.

I tried to be as clear and simple as possible. I also tried to align the style to the rest of the `readme`, hopefully that worked. I also quickly ran it through a spellcheck, but if you find anything wrong, feel free to edit it.

Quick question: I saw there is a package now that we could use as an image. I haven't tried it yet, is it ready for me to add it to the `readme` and `docker-compose`? For now, I wrote it with only having the `Dockerfile` option.

If you encounter any problems, feel free to ask and make adjustments as you like.

### CODE REVIEW

**Comment by chrivers** [comment]
Thank you very much - this is great!

I reviewed your work, and have a few notes. I hope you're up for another round :)

**Comment by ellwoodb** [comment]
Sure, no problem. Did you already comment on some things, because I can't see it?

**Review by chrivers — COMMENTED** [review]
Whoops - I misunderstood github's review system, so my comments were stuck in the pipeline 😅

File: README.md [hunk 1]
```diff
@@ -89,12 +98,55 @@ At this point, the server should start: (log timestamps omitted for clarity)
...
+The log output shows Bifrost talking with zigbee2mqtt, and finding some lights to control (office\_{1,2,3}).
 
 At this point, you're running a Bifrost bridge.
 
 The Philips Hue app should be able to find it on your network!
 
+### Docker
+
+> [!WARNING]
+> Docker support just recently got [merged](https://github.com/chrivers/bifrost/pull/2) to this project. It is not optimal yet and there are no prebuilt images (yet). If you encounter any bugs or have suggestions, feel free to leave your feedback [here](#problems-questions-feedback).
```
[the remark this answers is not in the document]
Comment (ellwoodb) [reply README.md:110]: Yes, that's way clearer :+1:

File: README.md [hunk 2]
```diff
@@ -10,11 +10,20 @@ might like to read the [comparison with DiyHue](doc/comparison-with-diyhue.md).
...
-To install Bifrost, you will need the following:
+There are currently two ways you can install Bifrost.
 
- 1. The rust language toolchain (https://rustup.rs/)
- 2. At least one zigbee2mqtt server to connect to
- 3. The MAC address of the network interface you want to run the server on
+1.  [Install manualy](#manual) from source (for now, this is the preferred method)
+2.  [Install it via Docker](#docker) (still WIP, some aspects may not be optimal for now)
+
+When you have these things available, you can download
```
[the remark this answers is not in the document]
Comment (ellwoodb) [reply README.md:18]: Ah yes, sorry that was just left over.

File: docker-compose.yaml [hunk 3]
```diff
@@ -1,7 +1,12 @@
...
     build: .
+    container_name: bifrost
+    restart: unless-stopped
+    network_mode: host
     volumes:
+      # If you followed the guide on our readme, you could also use relative paths:
+      # - ./config.yaml:/app/config.yaml
+      # - ./certs:/app/certs
       - /path/to/app-data/config.yaml:/app/config.yaml
       - /path/to/app-data/certs:/app/certs
```
[the remark this answers is not in the document]
Comment (ellwoodb) [reply docker-compose.yaml:12]: You're right. I was thinking about it, for when users don't need to clone the whole repo but just download the compose and config, but then relative paths are still more user-friendly I think.

Speaking of this, when the docker image is ready, we don't need the `git clone` instructions anymore. I would then maybe add a `curl` command to download the compose and config.

**Comment by chrivers** [comment]
Looking good!

Do you mind rebasing on top of master? I like to keep the git history reasonably clean :)

**Comment by ellwoodb** [comment]
Hey, not at Home right now, so I'll have to do that later.

I'm not very experienced with git and GitHub, do you mind pointing me in a direction on how to rebase / what you mean by that?

**Comment by chrivers** [comment]
Good news, I was able to rebase the branch from github :)

Thanks for the work, I'll merge it :+1:

**Comment by ellwoodb** [comment]
Perfect, glad to have helped! 👍

**Pull request merged**